src/image-base.ks

```diff
@@ -24,7 +24,9 @@ clearpart --initlabel --all --disklabel=gpt
 # See also coreos-growpart.service defined in fedora-coreos-base.yaml
 # You can change this partition layout, but note that the `boot` and `root`
 # filesystem labels are currently mandatory (they're interpreted by coreos-assembler).
-reqpart --add-boot
+reqpart
+# Keep explicitly labeled boot partition as it is not norm on all architectures.
+part /boot --size=300 --fstype="xfs" --label=boot
```

Comment:
I might be wrong, but does `aarch64` support `xfs` in the boot partition? @ausil?

Reply:
Hmm, I tried this before I think and ran into issues.  Let's test this with both `metal-bios` and `metal-uefi` on `x86_64` at least.

Reply:
@r4f4 I have just went back to what has been used in past> I will be happy to change it to anything that will work across the board.
Hopefully there will be no need to have per arch ks.

Reply:
I tested this and it works fine!  Going down to a 300M `/boot` instead of `1G` is also nice, we definitely don't need *that* much space there.